demo/apisix/tmail-apisix-plugin-runner/src/main/java/com/linagora/apisix/plugin/AppConfiguration.java

```diff
@@ -27,26 +28,43 @@ public class AppConfiguration {
     @Value("${redis.password}")
     private String redisPassword;
 
+    @Value("${redis.timeout:5000}")
+    private Integer redisTimeout;
+
+    @Value("${redis.ignoreErrors:false}")
```

Comment:
```suggestion
    @Value("${redis.ignoreErrors:true}")
```

Reply:
Default to false = no change logic when no configured

Benoit comment in related topic: https://github.com/linagora/james-project/issues/5175#issuecomment-2082169728

Reply:
You currently document this default to true cf https://github.com/linagora/tmail-backend/pull/1025/files#diff-e9f776705de13bbe1ae21988b098963eb25e1de36b3ee38bc0c07b10e44f03e5R47

That makes me confused.

> Benoit comment in related topic: https://github.com/linagora/james-project/issues/5175#issuecomment-2082169728

I think for James use case we need to be more careful as it has a community. While this is our plugin, I think it is OK to let it default to `true`.

Thoughts?

Reply:
Bith are fine to me. Let's go with `true`